sys-fs/copyfs/copyfs-1.0.1-r1.ebuild

```diff
@@ -30,13 +30,14 @@ PATCHES=(
 
 src_prepare() {
 	default
+	sed -i '/@/d' Makefile.in || die
```

Comment:
Explain!

Reply:
Whoops, that wasn't supposed to make it in!  It stopped the configure script from changing lines like `bindir=@bindir@`, but that doesn't matter as I just set `bindir` anyway when I called make.